dask/dataframe/tests/test_multi.py

```diff
@@ -560,24 +560,29 @@ def test_concat(join):
 
     kwargs = {"sort": False}
 
-    for (dd1, dd2, pd1, pd2) in [(ddf1, ddf2, pdf1, pdf2), (ddf1, ddf3, pdf1, pdf3)]:
-
-        expected = pd.concat([pd1, pd2], join=join, **kwargs)
-        result = dd.concat([dd1, dd2], join=join)
-        assert_eq(result, expected)
-
-    # test outer only, inner has a problem on pandas side
-    for (dd1, dd2, pd1, pd2) in [
-        (ddf1, ddf2, pdf1, pdf2),
-        (ddf1, ddf3, pdf1, pdf3),
-        (ddf1.x, ddf2.x, pdf1.x, pdf2.x),
-        (ddf1.x, ddf3.z, pdf1.x, pdf3.z),
-        (ddf1.x, ddf2.x, pdf1.x, pdf2.x),
-        (ddf1.x, ddf3.z, pdf1.x, pdf3.z),
-    ]:
-        expected = pd.concat([pd1, pd2], **kwargs)
-        result = dd.concat([dd1, dd2])
-        assert_eq(result, expected)
+    with warnings.catch_warnings():
```

Comment:
I don't think we want to change this. We need to ensure that users aren't seeing pandas warnings from Dask's usage of `pandas.concat`.

Reply:
Should we then filter warnings in `dd.concat`?

Reply:
No, I don't think so. We need to ensure that we call `pd.concat` with the correct parameters. I think that the test as written gets the behavior that we want.

Reply:
Removed the warnings filtering in tests, upon some digging found that the param wasn't being propagated to the functions being called in the code-flow.